src/System Application/App/AppSource Gallery/app.json

```diff
@@ -63,7 +63,7 @@
     ],   
     "internalsVisibleTo":  [
         {
-            "name": "Microsoft AppSource Product Gallery Test",
+            "name": "AppSource Product Gallery Test",
```

Comment:
still not pointing to a test library app

Reply:
correct, focus is bug fixing in this iteration.